src/api/app/views/webui/users/notifications/_notification_filter.html.haml

```diff
@@ -70,6 +70,11 @@
                                                               key: 'kind[appealed_decisions]', name: 'kind[]', value: 'appealed_decisions',
                                                               label_icon: 'fas fa-hand',
                                                               checked: selected_filter[:kind]&.include?('appealed_decisions') }
+    .px-4.py-1
+      = render partial: 'webui/shared/check_box', locals: { label: 'User Added/Removed on Groups', amount: counted_notifications['user_on_groups'],
```

Comment:
@saraycp The text looks still quite long in the filter area. Maybe we just call it `Group Roles`? or including it under the `Roles Granted` and `Roles Revoked` filters?

Reply:
What about `User on Groups`? It's independent of the roles

Reply:
Changed to `User on Groups`